Source/Virtusize.swift

```diff
@@ -135,6 +66,54 @@ public class Virtusize {
     /// A closure that is called when the operation fails
     typealias ErrorHandler = (VirtusizeError) -> Void
 
+	/// Todo
+	internal static var authToken: String? {
+		return UserDefaultsHelper.current.accessToken
+	}
+
+	/// The array of `VirtusizeView` that clients use on their mobile application
+	private static var virtusizeViews: [VirtusizeView] = []
+
+	/// The array of `VirtusizeView` that clients use on their mobile application
+	internal static var productTypes: [VirtusizeProductType]?
+
+	internal static var storeProduct: VirtusizeInternalProduct?
+	internal static var i18nLocalization: VirtusizeI18nLocalization?
+	internal static var bodyProfileRecommendedSize: BodyProfileRecommendedSize?
+	internal static var sizeComparisonRecommendedSize: SizeComparisonRecommendedSize?
+
+	/// The private property for product
+	private static var _product: VirtusizeProduct?
+	/// The Virtusize product to get the value from the`productDataCheck` request
+	public static var product: VirtusizeProduct? {
+		set {
+			guard let product = newValue else {
+				return
+			}
+			productCheck(product: product, completion: { product in
+				guard let product = product else {
+					return
+				}
+				_product = product
+				if let productId = _product!.productCheckData?.productDataId {
```

Comment:
Guard would also make sense here and would remove the nesting

Reply:
Done